readthedocs/core/utils/__init__.py

```diff
@@ -111,38 +111,9 @@ def trigger_build(project, version=None, record=True, force=False, basic=False):
 
 def send_email(recipient, subject, template, template_html, context=None,
                request=None):
-    '''
-    Send multipart email
-
-    recipient
-        Email recipient address
-
-    subject
-        Email subject header
-
-    template
-        Plain text template to send
-
-    template_html
-        HTML template to send as new message part
-
-    context
-        A dictionary to pass into the template calls
-
-    request
-        Request object for determining absolute URL
-    '''
```

Comment:
In this case, I was trying to avoid passing the request to the task. Having a task call a util function is a good pattern, and is more composable for testing, there just aren't any tests for composing emails yet. We could do util function -> task -> another util function, but seemed unnecessary.